src/scripts/templates/active/Active default template.js

```diff
@@ -36,6 +43,16 @@ function scanNode(as, msg) {
 	}
 }
 
+/**
+ * Scans a specific parameter in an HTTP message.
+ * 
+ * @param as the ActiveScan parent object that will do all the core interface tasks 
+ *     (i.e.: sending and receiving messages, providing access to Strength and Threshold settings,
+ *     raising alerts, etc.)
+ * @param msg the HTTP Message being scanned
+ * @param param the parameter being manipulated for this test/scan
```

Comment:
Could mention that it's the name of the parameter. Also, could indicate the type (string) per JSDoc? [1]

[1] http://usejsdoc.org/tags-param.html#names-types-and-descriptions